src/pages/TodayPage/Components/CreateReminder.tsx

```diff
@@ -152,8 +150,8 @@ function CreateReminder() {
             </Button>
             <Button
               color="white"
-              bg={!toDoData.title.trim() ? "red.300" : "red.500"}
-              disabled={!toDoData.title.trim()}
+              bg={!toDoData.title?.trim() ? "red.300" : "red.500"}
```

Comment:
Since we don't use onChange, we can not keep track of the name or description data inside the todoData. 
You should check always current value of them
```suggestion
              bg={!titleRef.current?.value.trim() ? "red.300" : "red.500"}
```